packages/components/Callout/macos/Callout.swift

```diff
@@ -289,15 +289,37 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 	private var proxyView: RCTView = RCTView()
 
 	private lazy var calloutWindow: CalloutWindow = {
-		let calloutWindowController = CalloutWindowRootViewController()
-		let window = CalloutWindow(contentViewController: calloutWindowController)
+		let window = CalloutWindow()
 		window.windowLifeCycleDelegate = self
 		window.isReleasedWhenClosed = true
+
 		window.styleMask = [.fullSizeContentView]
 		window.level = .popUpMenu
+		window.backgroundColor = .clear
+		window.isMovableByWindowBackground = false
+		
+		let visualEffect = NSVisualEffectView()
+		visualEffect.translatesAutoresizingMaskIntoConstraints = false
+		visualEffect.material = .menu
+		visualEffect.state = .active
+		visualEffect.wantsLayer = true
+		visualEffect.layer?.cornerRadius = 5.0
```

Comment:
Is this the default NSButton corner radius?

Reply:
Default menu corner radius, which I found by grabbing the macOS design templates and importing them into sigma: https://developer.apple.com/design/resources/

Reply:
I suppose I should pull that out as a constant since it's a magic number

Reply:
It's not easy to get the template into figma and find the corner radius, so here's a screenshot of me doing so. You can see "5" on the right

![MicrosoftTeams-image](https://user-images.githubusercontent.com/6722175/140189051-fab4356b-ecb5-4f1a-a937-231e534e27c4.png)